IFComp/lib/IFComp/Schema/Result/Comp.pm

```diff
@@ -206,6 +206,7 @@ __PACKAGE__->has_many(
 
 use DateTime::Moonpig;
 use Moose::Util::TypeConstraints;
+use List::Util qw(none);
```

Comment:
It doesn't look like this is used anywhere. Maybe it's left over from an earlier approach?